sysdeps/linux/generic/sysdeps.cpp

```diff
@@ -839,6 +839,28 @@ int sys_setpriority(int which, id_t who, int prio) {
 	return 0;
 }
 
+int sys_nice(int increment, int *new_nice) {
+	int current;
+	if (int e = sys_getpriority(0, 0, &current); e)
+		return e;
+
+	if (increment == 0) {
+		*new_nice = current;
+		return 0;
+	}
+
+	if (int e = sys_setpriority(0, 0, current + increment); e)
```

Comment:
It would be nice to note that the syscall silently clamps the value to the range; otherwise it might seem like clamping would be the sysdep's responsibility.